demo/app/layout/header.component.ts

```diff
@@ -1,4 +1,6 @@
-import { Component, Input} from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
+const defaultTheme = require('./../../../src/themes/default.theme.scss');
+const materialTheme = require('./../../../src/themes/material.theme.scss');
```

Comment:
this will include an additional style in the bundle size without using it, it would be nice to leave user import the required style manually

Reply:
@aitboudad it is used for demo app to be able to switch themes

Reply:
ahh sorry! I thought that was `src` code

Reply:
@varnastadeus is the material theme is optional only.

Reply:
@arun-maddheshia Yes, it will be optional.